wiki/en/Running-a-Server.md

```diff
@@ -298,6 +297,49 @@ You yourself should connect using the local network (LAN) address of the machine
 
 Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
 
+## DNS SRV record support
+
+Jamulus client supports [DNS SRV records ("service" records)](https://en.wikipedia.org/wiki/SRV_record). If Jamulus client finds an SRV record associated with the domain name given in the Connect window, it will try to connect to the endpoint host and port listed in the SRV record. This process is similar to HTTP redirection in a web browser, only it's between Jamulus client and the DNS server. Jamulus server is not involved in this interaction. Jamulus server just listens for connections on the same port listed in the SRV record.
+
+#### Why is this helpful?
+
+Jamulus client can connect to a server on a non-standard port by specifying the port as part of the server address. A server administrator may want to give users a simple address without the port details, while still serving Jamulus on a non-standard port. For this case, the port information can be acquired from an SRV record.
+
+For example, an address given in Jamulus client Connect window could look like ```jamulus.example.com```. If a DNS SRV record is found for this domain, Jamulus client will try to connect to Jamulus server on the host:port listed in the SRV record, such as ```jamulus.example.com:12345```.
+
+If no SRV records are found on the DNS server, Jamulus client will attempt to connect to Jamulus server as specified in the connection window.
+
+#### Creating SRV records
+
+SRV records are created by the administrator of the domain being used to host the Jamulus server. The SRV records are added through the administration portal (or API if available) of the domain's DNS hosting service. The format of the SRV record entry can vary by DNS hosting service but will generally look something like this.
+
+~~~
+  _service._proto.name. ttl IN SRV priority weight port target
+  
+  # or, more specifically
+  
+  _jamulus._udp.example.com. 60 IN SRV 0 5 12345 jamulus.example.com
+~~~
+
+**Tip:** You can host many Jamulus Servers at different ports on one host by giving each its own SRV record.
+
+See the documentation of your DNS provider for instructions on creating an SRV record for that provider.
+
+#### Example use case with SRV records
+
+This example assumes a DNS A or CNAME record resolves ```server1.example.com```, and ```server1.example.com``` has three Jamulus server instances, each listening on one of the three specified ports below.
```

Comment:
I think this should be rephrased as the grammar seems strange (I'm not a native speaker.)